### ISSUE TITLE
Overview

### ISSUE THREAD

**Issue by lilyminium** [whitead/dmol-book#26]
Hi @whitead, thanks for writing this book. It's neatly laid out and easy for beginners like me to understand. I hope you don't mind that I took LiveComSJ's tweet asking for feedback seriously.

# Overview

> Deep learning is specifically about *connecting* two types of data with a neural network **function**, which is differentiable and able to approximate any **function**. The classic example is *connecting* **function** and structure in molecules. 

Suggestion: The classic example is connecting molecular structure to its function.

Reason: "function" just meant a totally different thing in the sentence before. Reading "The classic example is connecting function" without the molecular context meant that I first thought of "connecting function" as a noun, i.e., a mapping from one space to another.

> **it’s** ability to generate new data.

its, no apostrosphe

> One example that sets deep learning apart from machine learning is in feature engineering.

I think many people would argue that deep learning is a form of machine learning. If I understand correctly, the next part of the paragraph argues that deep learning does *not* need feature engineering? Perhaps "One advantage that sets deep learning apart from other machine learning techniques is that it does not require feature engineering"?

> Previously training and using models in machine learning was a tedious process and required deriving equations for each model change.

Comma after previously?

> Deep learning is always a little tied-up in the implementation details. Thus, language choice can be a part of the learning process. In this book, we use Jax, Tensorflow, Keras, and scikit-learn for different purposes. 

It was a little odd for me that you jumped straight to libraries instead of saying Python first, considering that the section is titled "Language choice". Throughout this section you also refer to them all as languages. I would personally consider all of those to be "frameworks" rather than languages.

> Stackoverflow

Stack Overflow?

**Comment by whitead** [comment]
Thanks for your feedback! Very detailed and I agree with you on all points raised. I've addressed all your comments/suggestions in 829c6c7864ef576b24f541d7b2f53b423f6aba74.
